### PR TITLE
Update multi-region-setup.md

### PR DESCRIPTION
**Pull request by fabiog1901**
wrong indentation on list

### CODE REVIEW

File: v20.2/multi-region-setup.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ This page walks you through setting up a virtual environment for developing and
 
 1. Complete the previous section of the tutorial, [Create a Multi-Region Database Schema](multi-region-database.html).
 
-1. Make sure that you have the following installed on your local machine:
+2. Make sure that you have the following installed on your local machine:
 
-    - [CockroachDB](install-cockroachdb-mac.html)
-    - [Python 3](https://www.python.org/downloads/)
-    - [Pipenv](https://pipenv.readthedocs.io/en/latest/)
+- [CockroachDB](install-cockroachdb-mac.html)
```
Comment (rmloveland) [inline v20.2/multi-region-setup.md:15]: Hi Fabio!  This is still rendering weirdly in [the preview page](https://deploy-preview-9324--cockroachdb-docs.netlify.app/docs/v20.2/multi-region-setup.html).  To get this sublist to nest correctly, try indenting these bulleted links in by 4 spaces.

**Review by rmloveland — CHANGES_REQUESTED** [review]
Closer but I think the indenting needs a tweak!  (PS thanks for the PR!!!!1!)

**Comment by jseldess** [comment]
@rmloveland, can you please take this one over? You may need to open a fresh PR at this point.

**Review by fabiog1901 — COMMENTED** [review]
Hi Rich, trying to figure out how to accept your suggestions so we can move forward..

**Comment by rmloveland** [comment]
> can you please take this one over? You may need to open a fresh PR at this point.

Sure!

**Comment by rmloveland** [comment]
In the meantime, this appears to be fixed everywhere now as of this writing

- https://www.cockroachlabs.com/docs/v20.1/multi-region-setup.html
- https://www.cockroachlabs.com/docs/v20.2/multi-region-setup.html
- https://www.cockroachlabs.com/docs/v21.1/movr-flask-setup.html
- https://www.cockroachlabs.com/docs/v21.2/movr-flask-setup.html

Thanks for the PR @fabiog1901 - sorry for my delay.  This is now fixed at least :-)

**Pull request closed without merging**